303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/handlers/airline/NativePostgresqlAirlineHandlerIT.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.qe.vertx.sql.handlers.airline;
+
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+import io.quarkus.qe.vertx.sql.test.profiles.NativeProfile;
+import io.quarkus.test.junit.NativeImageTest;
+
+@NativeImageTest
+@EnabledIfSystemProperty(named = NativeProfile.QUARKUS_PROFILE, matches = NativeProfile.NATIVE)
```

Comment:
With `NativeImageTest` annotation, this other precondition is not required: `@EnabledIfSystemProperty(named = NativeProfile.QUARKUS_PROFILE, matches = NativeProfile.NATIVE)`

Reply:
It is required as now we're configuring the failsafe Maven plugin to be run always. So, we need to exclude the Native ITs using this system property. You can try yourself by removing this line and running `mvn clean verify`. It should fail when trying to run this test because the native path is not configured.
I'm waiting for your confirmation about this.